rules/no-hex-escape.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fix = value => {
+	if (!/\\x[a-fA-F0-9]{2}/.test(value)) {
```

Comment:
Assign `/\\x[a-fA-F0-9]{2}/` to a top-level variable. That way, the regex won't get recreated at every iteration (and every time the linter encounters a Literal).